link/src/main/java/com/stripe/android/link/repositories/LinkApiRepository.kt

```diff
@@ -235,18 +234,20 @@ internal class LinkApiRepository @Inject constructor(
                         publishableKeyProvider(),
                         stripeAccountIdProvider()
                     )
-                )?.paymentDetails?.first()?.let {
+                )?.paymentDetails?.first()?.let { paymentDetails ->
```

Comment:
It looks like there's some new behavior in these first few link classes. Can we add some tests?

Reply:
No new behavior, just pulling this code out of Link’s `ConfirmStripeIntentParamsFactory#createPaymentMethodCreateParams`.